src/main/java/net/coderbot/iris/Iris.java

```diff
@@ -102,6 +124,24 @@ private void loadInternalShaderpack() {
 		logger.info("Using internal shaders");
 	}
 
+	public static void reload() throws IOException {
+		//currently this first line can be used to reload to a diff shaderpack, but it should be removed
+		//when there is a gui or a better system for changing packs at runtime
+		//or could be kept for the gui to use
+		irisConfig.initialize();
+		Path shaderPackRoot;
+		if (!irisConfig.isInternal()) {
+			shaderPackRoot = shaderpacksDirectory.resolve(irisConfig.getShaderPackName());
+		} else {
+			shaderPackRoot = FabricLoader.getInstance().getModContainer("iris").orElseThrow(() -> new RuntimeException("Failed to get the mod container for Iris!"))
+					.getRootPath()
+					.resolve("shaders");
+		}
+		ShaderPack pack = new ShaderPack(shaderPackRoot.resolve("shaders"));
+		pipeline = new ShaderPipeline(pack);
+		currentPack = pack;
```

Comment:
I'm going to assume that this code is duplicated, I would prefer for it to not be duplicated